makefile-sysdig

```diff
@@ -1,16 +1,18 @@
 # This repo is a fork of a public upstream, so Sysdig specific make commands have been added in a separate makefile
 
-DOCKER := docker
 ARTIFACTORY := docker.internal.sysdig.com
-IMAGE_REPO := docker-bench-dependency
+PROJECT := docker-bench
+IMAGE_REPO := $(PROJECT)-dependency
 IMAGE_TAG?= dev
 DEPENDENCY_IMAGE_NAME := $(ARTIFACTORY)/$(IMAGE_REPO):$(IMAGE_TAG)
 
-build-dependency-image:
-	$(DOCKER) build -t $(DEPENDENCY_IMAGE_NAME) -f Dependency-Dockerfile .
-
-push-dependency-image:
-	$(DOCKER) push $(DEPENDENCY_IMAGE_NAME)
+ifndef PUSH
+	PUSH_FLAG := ""
+else
+	PUSH_FLAG := "--push"
+endif
 
-delete-dependency-image:
-	$(DOCKER) rmi $(DEPENDENCY_IMAGE_NAME)
```

Comment:
Do we not need the delete target anymore? Do we clean up images on the builder another way?

Reply:
> Do we not need the delete target anymore? Do we clean up images on the builder another way?

I don't believe anything else is needed for cleanup. buildx just builds, images aren't "loaded" into the docker graph dir unless requested (and we aren't). The removal/creation of the buildx worker container each time is all we need (AFAIK).

Note that I considered a multibranch approach where builders would be namespaced and could run independently, but decided wasn't critical.